pages/common/docker-compose.md

```diff
@@ -0,0 +1,27 @@
+# docker-compose
+
+> Run and manage multi container docker applications.
+
+- Create and start all containers in the background using a local `docker-compose.yml` file:
```

Comment:
Maybe better to use the "current directory" than "local". Makes it much more clear.

Reply:
agreed